airflow/models/dagbag.py

```diff
@@ -416,6 +427,19 @@ def collect_dags(
                              format(dag_names),
                              file_stat.duration)
 
+    def collect_dags_from_db(self):
+        """Collects DAGs from database."""
+        start_dttm = timezone.utcnow()
+        # DAG post-pcocessing steps such as self.bag_dag and croniter are not needed as
+        # they are done by scheduler before serialization.
+        # The dagbag contains all rows in serialized_dag table. Deleted DAGs are deleted
+        # from the table by the scheduler job.
+        self.log.info("Filling up the DagBag from database")
+        self.dags = SerializedDagModel.read_all_dags()
+        Stats.gauge(
```

Comment:
Can we use `Stats.timing` here? And you can just pass `timedelta` in.

Reply:
> Can we use `Stats.timing` here? And you can just pass `timedelta` in.

Good point! Done.